crates/ironrdp-web/src/clipboard.rs

```diff
@@ -601,14 +602,6 @@ impl ClipboardData {
         self.items.push(item);
     }
 
-    pub(crate) fn items(&self) -> &[ClipboardItem] {
```

Comment:
Removed these methods:
* `pub(crate) fn is_empty(&self) -> bool`
Not used.
*  `pub(crate) fn items(&self) -> &[ClipboardItem]` 
It's not used, but the same method from `iron_remote_desktop::ClipboardData` trait, which is implemented for the struct.